src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/IsDefendantLinkedTest.java

```diff
@@ -22,7 +23,8 @@
 
 
     @Test
-    public void shouldReturn200HttpStatusAndStatusTrueWhenClaimFoundAndIsLinked() throws Exception {
+    @Ignore("Disabled due to using CCD")
```

Comment:
When this will be enabled?

Reply:
same as above this shouldn't be disabled

Reply:
Would i not have to disable this as it is using V1?
We have the same test implemented but using v2 below